NDTensors/ext/NDTensorsAMDGPUExt/linearalgebra.jl

```diff
@@ -0,0 +1,66 @@
+# AMDGPU.jl does not provide these yet
+struct JacobiAlgorithm end
+struct QRAlgorithm end
+struct ROCtoCPUSVD end
+
+JACOBI_MAX_ITERATIONS = 500
+
+function NDTensors.svd_catch_error(A::ROCMatrix; alg::String="jacobi_algorithm")
+  if alg == "jacobi_algorithm"
+    alg = JacobiAlgorithm()
+  elseif alg == "qr_algorithm"
+    alg = QRAlgorithm()
```

Comment:
Is `QRAlgorithm` supported for `ROCMatrix`? If not this should be removed.

Reply:
It is supported by `rocSOLVER`, and I have written the wrapper for it in `AMDGPU.jl` which I will put in a future PR. But I'm happy to take this out for now of course.